src/vtol.py

```diff
@@ -67,11 +67,52 @@ def __init__(self, *args): #pylint: disable=useless-super-delegation
     coms = None
 
     # pylint: disable=no-self-use
-    def coms_callback(self, message, _):
+    def coms_callback(self, message):
         '''callback for radio messages'''
-        # TODO respond to xbee messagge
-        data = json.loads(message.data)
-        print(data['type'])
+        #message will be an object, we will be responding to the type attribute
+        #makes the message a dictionary
+        parsed_message = json.loads(message.data)
+        #gives us the specific command we want the drone to executre
+        command = parsed_message['type']
+
+        print(parsed_message['type'], type(parsed_message['type']))
+
+        #checking for valid command
+        if command not in ("takeoff", "land"):
+            raise Exception("Error: Unsupported status for vehicle")
```

Comment:
I like this, maybe make a tuple containing "takeoff" and "land" so this doesn't get cluttered later on